src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json.schemastore.org/dotnetcli.host",
+  "symbolInfo": {
+    "skipRestore": {
```

Comment:
Just confirming whether you've had a chance to test out these options on the CLI? Notably, the `Hosted` option is working appropriately (ie. creating empty blazor hosted vs empty blazor standalone).